back-end/helpers/test-helpers/testIntentGenerator.js

```diff
@@ -0,0 +1,74 @@
+export default class testIntentGenerator {
+    generateOneIntentModel() {
+        return [
+            { question: "Waluigi and Wario often say something-DOT- What is it?",
+                children: {"The brothers Waluigi and Wario have their catchphrase 'waaa'-DOT-": 1},
+                total_children: 1,
+                project_id: 1
+            }
+        ];
+    };
+
+    generateOneAnalyzedIntent() {
+        return [
+            { question: "Waluigi and Wario often say something. What is it?",
+                children: {"The brothers Waluigi and Wario have their catchphrase 'waaa'.": 1}
+            }
+        ];
+    };
```

Comment:
```suggestion
    generateAnalyzedIntent(intent) {
        return {question, children}
    };
```
(Marking as pseudo code)